spec/generator.json

```diff
@@ -31,10 +31,19 @@
             ]
         },
 
+        "file": {
+            "description": "Represents a source file",
+            "fields": [
+                { "name": "name", "type": "string", "description": "The recommended name for the file, including the path to the file, where appropriate.", "example": "com/myorg/myapplication/Foo.java" },
```

Comment:
I was thinking of splitting the 'name' field into 'name' and an optional 'path' field. I think this is more self-documenting and easier to consume.